SPECS/node-problem-detector/002-remove_windows_build.patch

```diff
@@ -26,5 +27,7 @@ diff -Naur a/Makefile b/Makefile
 -		./test/e2e/$(subst -,,$*)
 -
  output/linux_amd64/bin/%: $(PKG_SOURCES)
- 	GOOS=linux GOARCH=amd64 CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on go build \
- 		-mod vendor \
```

Comment:
Why is removing `-mod vendor` necessary now as part of removing Windows build options?

Reply:
This is unmodified trailing source code from the patch that github is highlighting as a diff between the old patch and new patch rather than something i changed myself. 

"-mod vendor" is on the following line in the source code but appears to have been removed because of the confusing highlighting and patch generation.